examples/samples/partitionedQueues.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  delay,
+  SendableMessageInfo,
+  QueueClient,
+  ReceiveMode,
+  generateUuid,
+  Namespace,
+  ReceiveHandler,
+  OnError,
+  OnMessage,
+  ServiceBusMessage,
+  MessagingError
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log(`str: ${str}`);
+console.log(`path: ${path}`);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createQueueClient(path, { receiveMode: ReceiveMode.peekLock });
+
+  // populate the queue with messages
+  await sendMessages(client);
+
+  let rcvHandler: ReceiveHandler;
```

Comment:
All other samples have the receiving part of the sample in a separate function. Can we do the same here to maintain consistency?

Reply:
Not all the samples. I think I followed this convention from SendersReceiversWithQueues sample and kept the receiving part in the main module. 
Shall I create a separate function ?

Reply:
I meant all other samples in this repo

Reply:
Okay, I'll change this :)